docs/docs/deploying-to-now.md

```diff
@@ -2,36 +2,67 @@
 title: Deploying to Now
 ---
 
-In order to deploy your Gatsby project using [Now](https://zeit.co/now), you can do the following:
+[ZEIT Now](https://zeit.co/now) is a [cloud platform for serverless deployment](https://zeit.co/docs/v2/getting-started/introduction-to-now/) that you can use to deploy your Gatsby projects and [alias them](https://zeit.co/docs/v2/domains-and-aliases/aliasing-a-deployment/) to your domain or a free `.now.sh` suffixed URL.
 
-1.  Install the Now CLI
+This guide will show you how to get started in a few quick steps:
 
-`npm install -g now`
+## Step 1: Getting Started with Gatsby
+If you haven't already [set up a Gatsby project](https://www.gatsbyjs.org/docs/quick-start) you can do so by first installing Gatsby globally:
 
-2.  Run `gatsby build` at the root of your project.
+```shell
+npm install --global gatsby-cli
+```
+
+Then generate a project with the following command:
+```shell
+gatsby new <your project name>
+```
 
-3.  Run `now` inside `public/`. This will upload your project to the cloud.
+## Step 2: Getting Now
+
+You can use Now by installing [Now Desktop](https://zeit.co/docs/v2/getting-started/installation/#now-desktop), which also installs Now CLI and keeps it up-to-date automatically.
+
+To install Now CLI quickly with npm, use the following:
+```shell
+npm install -g now
+```
 
-For large project sizes, it is better to install a Node server package (such as `serve`, or `http-server`):
+## Step 3: Preparing to Deploy
 
-1.  Install a Node server package
+With Now CLI installed, we can go on to deploy our previously setup Gatsby project by first creating a `now.json` file with the following contents:
 
-`npm install --save serve`
+```json:title=now.json
+{
+  "version": 2,
+  "name": "my-gatsby-project",
+  "builds": [
+    { "src": "package.json", "use": "@now/static-build", "config": {"distDir": "public"} }
+  ]
+}
+```
 
-2.  Add a `start` script to your `package.json` file. This is what Now will use to run your application:
+This will allow Now to understand that we want to use the [latest Now 2.0 version](https://zeit.co/blog/now-2) of [the platform](https://zeit.co/docs/v2/platform/overview/), to set the project name to be `my-gatsby-project`, and to use the [@now/static-build builder](https://zeit.co/docs/v2/deployments/official-builders/static-build-now-static-build/) to take the `package.json` file as an entrypoint and use the `public` directory as the directory our final content will end up at.
```

Comment:
Can we re-visit this? It's quite a sentence. I think it may be helpful to break this out into a list, e.g.

```suggestion
This `now.json` file will allow us to do several things, specifically:

- Use the [latest Now 2.0 version](https://zeit.co/blog/now-2) of [the platform](https://zeit.co/docs/v2/platform/overview/)
- Set the project name to `my-gatsby-project`
- Use the [@now/static-build builder](https://zeit.co/docs/v2/deployments/official-builders/static-build-now-static-build/) to take the `package.json` file as an entrypoint and use the `public` directory as the our content directory
```